java/connector-framework-internal/src/main/java/org/identityconnectors/framework/impl/serializer/CommonObjectHandlers.java

```diff
@@ -497,27 +476,23 @@ public void serialize(final Object object, final ObjectEncoder encoder) {
 
             @Override
             public Object deserialize(final ObjectDecoder decoder) {
-                @SuppressWarnings("unchecked")
-                final Set<ObjectClassInfo> objectClasses =
+                @SuppressWarnings("unchecked") final Set<ObjectClassInfo> objectClasses =

```

Comment:
Please avoid such formatting-only changes.

Reply:
Thank you, I guess same as the later ones, Ill have this fixed.